js/testapps/flow-simple-ai/src/index.ts

```diff
@@ -908,21 +909,24 @@ ai.defineFlow(
     name: 'geminiImages',
     inputSchema: z.string().optional(),
   },
-  async (setting) => {
-    const { message } = await ai.generate({
-      model: gemini20FlashExp,
-      prompt: `Generate a choose your own adventure intro${setting ? ` in ${setting}` : ''}, then generate a first-person image as if I'm in the story.`,
+  async (setting, { sendChunk }) => {
+    const { response, stream } = ai.generateStream({
+      model: googleAI.model('gemini-2.0-flash-preview-image-generation'),
```

Comment:
Is this model long running?

Reply:
No. Just updated an unrelated example to do streaming.